pkg/termcorder/termcorder_test.go

```diff
@@ -0,0 +1,116 @@
+package termcorder_test
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/haguro/termcord/pkg/termcorder"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTermcording(t *testing.T) {
+	t.Parallel()
+	cfg := &termcorder.Config{}
+	tc, err := termcorder.NewTermcording(cfg)
+	assert.IsType(t, &termcorder.Termcording{}, tc)
+	assert.NoError(t, err)
+}
+
+func TestTermcordingFromFlags(t *testing.T) {
+	t.Parallel()
+	t.Run("Test setting flags", func(t *testing.T) {
+		os.Args = []string{"./termcord", "-h", "-q", "-k", "foo.txt", "bar", "baz"}
```

Comment:
We definitely shouldn't set `os.Args`—that's very bad manners!

Reply:
I agree that it feels a bit dirty. Any suggestions as to how we might test `TermcordingFromFlags` without writing to `os.Args`?

Reply:
`TestMain` might be one way to do this, though not necessarily the best.

Reply:
True. Anyway having looked just a little closer I realise that my question was a little silly. `TermcordingFromFlags` could simply take an `args` slice. To which we can pass `os.Args` from `main` and an arbitrary args slice from tests.